server/sonar-web/src/main/js/components/shared/drilldown-link.js

```diff
@@ -47,12 +47,12 @@ const ISSUE_MEASURES = [
   'new_vulnerabilities'
 ];
 
-export const DrilldownLink = React.createClass({
-  isIssueMeasure() {
+export class DrilldownLink extends React.PureComponent {
+  isIssueMeasure = () => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)